<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Stage:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Sequence Name:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stage_comboBox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Sequence stage</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="sequence_name_lineEdit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Sequence code name</string>
          </property>
          <property name="maxLength">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="main_verticalLayout"/>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QCheckBox" name="select_all_checkBox">
     <property name="toolTip">
      <string>Select or deselect all shots</string>
     </property>
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="new_shot_pushButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Create new shot</string>
        </property>
        <property name="text">
         <string>New Shot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refresh_shots_pushButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Refresh shot list</string>
        </property>
        <property name="text">
         <string>Refresh Shots</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ubercam</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="ubercam_pushButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Create Ubercam</string>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove_ubercam_pushButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remove Ubercam</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="look_thru_ubercam_pushButton">
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>23</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Look through Ubercam</string>
        </property>
        <property name="text">
         <string>Look Thru</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Batch</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="playblast_sequence_pushButton">
        <property name="toolTip">
         <string>Playblast all selected shots</string>
        </property>
        <property name="text">
         <string>Playblast</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="break_out_selected_pushButton">
        <property name="toolTip">
         <string>Break out all selected shots</string>
        </property>
        <property name="text">
         <string>Break Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
